config/webpack/production.js

```diff
@@ -1,3 +1,12 @@
 const environment = require('./environment')
+const { merge } = require('webpack-merge')
+
+const sassLoader = environment.loaders.get('sass');
+
+sassLoader.use.map(function(loader) {
+  if (loader.loader === "css-loader") {
```

Comment:
While I think we'd want sourcemaps in development, I'm still trying to discern between `css-loader` and `sass-loader` to the effect of: Is `css-loader` doing anything for us at all?

In the upcoming Webpacker 6+, it's conditionally opted into, but not listed as a requirement for Sass usage:

https://github.com/rails/webpacker#sass

https://github.com/rails/webpacker/blob/6efb3ee/package/environments/base.js#L66-L78

In other words, it might not even be an issue after upgrading?

Reply:
I moved it to `environment.js`, but wasn't sure how to confirm it doesn't mess up development?

Reply:
> I moved it to `environment.js`, but wasn't sure how to confirm it doesn't mess up development?

We're not currently loading any CSS via Webpacker since #4587. My expectation would be that if we were to revert 298f8f9 (even locally, temporarily in this branch), any problems would be obvious either in (a) failing to compile or (b) styles not being present in the document capture step.

Reply:
I attempted reverting https://github.com/18F/identity-idp/commit/298f8f928c8ec41c60e8bd4c950cae2603224267 in 585fa32 and didn't have any issues. The separate CSS pack is loaded as expected.